com_dpfields/admin/helpers/dpfields.php

```diff
@@ -187,6 +197,14 @@ public static function getFields ($context, $item = null, $prepareValue = false)
 						));
 					}
 
+					/*
+					 * On after field render
+					 * Event allow plugins to modfify the output of the prepared field
+					 */
+					if ($value)
```

Comment:
But IMO if you have a *before* event also a *after* event should be triggered. Plugin developers will expect that. This needs also to be changed in com_fields.